Jenkinsfile

```diff
@@ -48,13 +48,24 @@ pipeline {
         }
 
         stage('Publish') {
-            when { branch 'master'}
+            when {
+                expression {!env.BRANCH_NAME.startsWith('PR')}
+            }
             steps {
                 sh "docker login quay.io -u ${QUAY_IO_CREDS_USR} -p ${QUAY_IO_CREDS_PSW}"
                 sh "podman login quay.io -u ${QUAY_IO_CREDS_USR} -p ${QUAY_IO_CREDS_PSW}"
                 sh "make publish"
             }
         }
+
+        stage('Publish master') {
+            when { branch 'master'}
+            steps {
+                sh "docker login quay.io -u ${QUAY_IO_CREDS_USR} -p ${QUAY_IO_CREDS_PSW}"
```

Comment:
I think we can move the login part to the `INIT` stage. I wouldn't want to fail after an hour just because we couldn't login to the registry.